### ISSUE TITLE
CLI Commands: bash completion support

### ISSUE THREAD

**Issue by tanaka-takayoshi** [aspnet/EntityFramework#5976]
### Steps to reproduce
Installing Microsoft.EntityFrameworkCore.Tools

### The issue
Describe what is not working as expected.
``dotnet ef`` command doesn't support bash completion. Do you have any plan to implement it?
For example, azure clit (xaplt-azure-cli) supports it.
https://github.com/azure/azure-xplat-cli#configure-auto-complete

### Further technical details

EF Core version: 1.0.0-preview2-final (tools)
Operating system: Linux, OSX
Visual Studio version: n/a

Other details about my project setup:

**Comment by dennisroche** [comment]
I think this would be useful and following the work already done for `dotnet` tab completion, https://github.com/dotnet/cli/blob/master/Documentation/general/tab-completion.md

**Comment by bricelam** [comment]
I don't think it would be cheap, they switched to a different parser. But I'm not opposed to investigating the new one.
